src/test/java/com/google/cloud/resourcemanager/ResourceManagerImplTest.java

```diff
@@ -454,4 +496,185 @@ public void testRuntimeException() {
     thrown.expectMessage(exceptionMessage);
     resourceManagerMock.get(PARTIAL_PROJECT.getProjectId());
   }
+
+  @Test
+  public void testClearOrgPolicy() {
+    try {
+      RESOURCE_MANAGER.clearOrgPolicy(RESOURCE, ORG_POLICY_INFO);
+      fail("Should fail because the organization policy doesn't exist.");
+    } catch (ResourceManagerException expected) {
+      assertEquals(404, expected.getCode());
+      assertTrue(expected.getMessage().contains("Not Found"));
+    }
+  }
+
+  @Test
+  public void testGetEffectiveOrgPolicy() {
+    ResourceManagerRpcFactory rpcFactoryMock = EasyMock.createMock(ResourceManagerRpcFactory.class);
```

Comment:
new code should be using Mockito:

https://engdoc.corp.google.com/eng/doc/devguide/java/testing/mocking.md?cl=head

This applies even if existing tests use EasyMock.

Reply:
done